packages/gatsby/src/utils/api-node-helpers-docs.js

```diff
@@ -0,0 +1,293 @@
+   *     type: `TypeOfNode`,
+   *     contentDigest: createContentDigest(nodeData)
+   *   }
+   * }
+   */
+  createContentDigest: true,
+
+  /**
+   * All action creators wrapped with a dispatch.
+   *
+   * See [`actions`](/docs/actions/) reference.
+   * @type {Actions}
+   * @deprecated Will be removed in gatsby 3.0. Use [actions](#actions)
+   * instead.
+   */
+  boundActionCreators: true,
+
+  /**
+   * All action creators wrapped with a dispatch.
+   *
+   * See [`actions`](/docs/actions/) reference.
+   * @type {Actions}
+   */
+  actions: true,
+
+  /**
+   * @param {Node} node
+   * @returns {Promise<string>}
+   * @example
+   * module.exports = async function onCreateNode(
+   *   { node, loadNodeContent, actions, createNodeId }
+   * ) {
+   *   if (node.internal.mediaType === 'text/markdown') {
+   *     const { createNode, createParentChildLink } = actions
+   *     const textContent = await loadNodeContent(node)
+   *     // process textContent and create child nodes
+   *   }
+   * }
+   */
+  loadNodeContent: true,
+
+  /**
+   * Internal redux state used for application state. Do not use, unless you
+   * absolutely must. Store is considered private API and can change with
+   * any version.
+   * @type {Redux.Store}
+   */
+  store: true,
+
+  /**
+   * Internal event emitter / listener.  Do not use, unless you absolutely
+   * must. Emitter is considered private API and can change with any version.
+   * @type {Emitter}
+   */
+  emitter: true,
+
+  /**
+   * Get array of all nodes.
+   * @returns {Node[]} Array of nodes.
+   * @example
+   * const allNodes = getNodes()
+   */
+  getNodes: true,
+
+  /**
+   * Get single node by given ID.
+   * Don't use this in graphql resolvers - see
+   * [`getNodeAndSavePathDependency`](#getNodeAndSavePathDependency).
+   * @param {string} ID id of the node.
+   * @returns {Node} Single node instance.
+   * @example
+   * const node = getNode(id)
+   */
+  getNode: true,
+
+  /**
+   * Get array of nodes of given type.
+   * @param {string} Type of nodes
+   * @returns {Node[]} Array of nodes.
+   * @example
+   * const markdownNodes = getNodesByType(`MarkdownRemark`)
+   */
+  getNodesByType: true,
+
+  /**
+   * Compares `contentDigest` of cached node with passed value
+   * to determine if node has changed.
+   * if node changed.
+   * @param {string} id of node
+   * @param {string} contentDigest of node
+   * @returns {boolean}
+   * @deprecated This check is done internally in Gatsby and it's not necessary to use it in plugins. Will be removed in gatsby 3.0.
+   */
+  hasNodeChanged: true,
+
+  /**
+   * Set of utilities to output information to user
+   * @type {GatsbyReporter}
+   */
+  reporter: true,
+
+  /**
+   * Get single node by given ID and creates dependency for given path.
+   * This should be used instead of `getNode` in graphql resolvers to enable
+   * tracking dependencies for query results. If it's not used Gatsby will
+   * not rerun query if node. See
```

Comment:
missing end of sentence? _if node `<something>`._

Reply:
Finished sentence in https://github.com/gatsbyjs/gatsby/pull/12087/commits/c13fdef4d8625f7596ed133d9057e1b52a926ad4